mne/channels/channels.py

```diff
@@ -1335,3 +1335,63 @@ def _get_ch_info(info):
     return (has_vv_mag, has_vv_grad, is_old_vv, has_4D_mag, ctf_other_types,
             has_CTF_grad, n_kit_grads, has_any_meg, has_eeg_coils,
             has_eeg_coils_and_meg, has_eeg_coils_only)
+
+
+def create_1020_montage_mapping(ch_names, midline="z12"):
+    r"""Return dict mapping from ROI names to lists of picks for 10/20 setups.
+
+    This passes through all channel names, and uses a simple heuristic to
+    separate channel names into three Regions of Interest: Left, Midline and
+    Right. The heuristic is that channels ending on any of the characters in
+    `midline` are filed under that heading, otherwise those ending in uneven
+    numbers to the left, those in even numbers to the right. Other channels
+    are ignored. This is appropriate for 10/20 files, but not for other channel
+    naming conventions.
+    If an info object is provided, lists are sorted from posterior to anterior.
```

Comment:
I'd prefer sorting from anterior to posterior, but if you think the other way is better then no worries.

Reply:
Maybe it's badly described ... the way it works now, if you use the sorting to index into an array and heatmap it, you get plots where the posterior channels are at the bottom (as in the examples).